cpp/src/parquet/metadata.cc

```diff
@@ -19,6 +19,8 @@
 
 #include <algorithm>
 #include <cinttypes>
+#include <csignal>
+#include <iostream>
```

Comment:
Why are `csignal` and `iostream` needed here?

Reply:
Removed.